app/src/main/java/com/sopt/now/compose/ui/ch_password/ChPasswordScreen.kt

```diff
@@ -0,0 +1,160 @@
+    var newPw by remember { mutableStateOf("") }
+    var verifyPw by remember { mutableStateOf("") }
+
+    val authState by viewModel.liveData.observeAsState()
+
+    LaunchedEffect(authState) {
+        authState?.let { state ->
+            snackbarHostState.showSnackbar(
+                message = state.message,
+                duration = SnackbarDuration.Short
+            )
+            if (state.isSuccess) {
+                navController.navigate("login") {
+                    popUpTo("chPassword") { inclusive = true }
+                }
+            }
+        }
+    }
+
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
+        snackbarHost = { SnackbarHost(hostState = snackbarHostState) },
+        bottomBar = {}
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(innerPadding)
+                .padding(horizontal = 20.dp, vertical = 20.dp),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Text(
+                stringResource(R.string.ch_pw),
+                fontSize = 30.sp,
+            )
+
+            Spacer(Modifier.height(20.dp))
+            OutlinedTextField(
+                value = prePw,
+                onValueChange = { prePw = it },
+                label = { Text(stringResource(R.string.pre_pw)) },
+                singleLine = true,
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Next,
+                    keyboardType = KeyboardType.Password
+                ),
+                visualTransformation = PasswordVisualTransformation(),
+                modifier = Modifier.fillMaxWidth()
+            )
+
+            Spacer(Modifier.height(16.dp))
+            OutlinedTextField(
+                value = newPw,
+                onValueChange = { newPw = it },
+                label = { Text(stringResource(R.string.new_pw)) },
+                singleLine = true,
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Next,
+                    keyboardType = KeyboardType.Password
+                ),
+                visualTransformation = PasswordVisualTransformation(),
+                modifier = Modifier.fillMaxWidth()
+            )
+
+            Spacer(Modifier.height(16.dp))
+            OutlinedTextField(
+                value = verifyPw,
+                onValueChange = { verifyPw = it },
+                label = { Text(stringResource(R.string.verify_pw)) },
+                singleLine = true,
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Next,
+                    keyboardType = KeyboardType.Password
+                ),
+                visualTransformation = PasswordVisualTransformation(),
+                modifier = Modifier.fillMaxWidth()
+            )
+
+            Spacer(modifier = Modifier.weight(1f))
+            CustomBtn(
```

Comment:
```suggestion
            CustomButton(
```

- 축약어 지양해주세요
- Cusotm이라고 하면 어떤 것이 떠오르시나요? 모호한 용어는 소통을 모호하게 만듭니다. 어떤 특성을 가진 버튼인지 좀 더 정확히 적어주면 좋을 것 같아요. 단 디자인 시스템 컴포넌트에 정의된 버튼이다 라고 하시면 {프로덕트이름}{컴포넌트 이름}으로 보통 적더라구요.

Ex) [SoptampButton](https://github.com/sopt-makers/sopt-android/blob/develop/feature/soptamp/src/main/java/org/sopt/official/stamp/designsystem/component/button/SoptampButton.kt)